lib/api/core/plugins/pluginsManager.js

```diff
@@ -473,295 +636,269 @@ function initControllers (controllers, routes, plugin, pluginName) {
 
       if (typeof description[action] !== 'string' || description[action].length === 0) {
         // eslint-disable-next-line no-console
-        console.error(`[Plugin Manager] Error loading ${pluginName}: invalid action description (expected non-empty string): ${action}`);
+        console.warn(clc.yellow(`${errorControllerPrefix} Invalid action description (expected non-empty string, got: "${clc.bold(typeof action)}")`));
         return false;
       }
 
       if (!plugin.object[description[action]] || typeof plugin.object[description[action]] !== 'function') {
         // eslint-disable-next-line no-console
-        console.error(`[Plugin Manager] Error loading ${pluginName}: action ${pluginName}.${description[action]} is not a function`);
+        console.warn(clc.yellow(`${errorControllerPrefix} Action "${clc.bold(plugin.name + '.' + description[action])}" is not a function`));
         return false;
       }
 
-      if (!controllers[`${pluginName}/${controller}`]) {
-        controllers[`${pluginName}/${controller}`] = {};
+      if (!pluginsManager.controllers[`${plugin.name}/${controller}`]) {
+        pluginsManager.controllers[`${plugin.name}/${controller}`] = {};
       }
 
-      controllers[`${pluginName}/${controller}`][action] = plugin.object[description[action]].bind(plugin.object);
+      pluginsManager.controllers[`${plugin.name}/${controller}`][action] = plugin.object[description[action]].bind(plugin.object);
       return true;
     });
   });
+
   const allowedVerbs = ['get', 'head', 'post', 'put', 'delete'];
 
   if (!controllerImported) {
```

Comment:
no need to use the variable `controllerImported` anymore if we throw an exception when the controller cannot be imported